src/app/pages/send/send-crypto-asset-form/form/btc/btc-send-form.tsx

```diff
@@ -42,15 +42,14 @@ export function BtcSendForm() {
     utxos,
     validationSchema,
   } = useBtcSendForm();
-
   return (
     <Box width="100%" pb="base">
       <Formik
         initialValues={createDefaultInitialFormValues({
           ...routeState,
           recipientBnsName: '',
         })}
```

Comment:
In the original solution, I discovered the form was not re-running validation `onSubmit`. I think this is as the `chooseTransactionFee` `onSubmit` handler was not run `async`.